xbmc/pvr/addons/PVRClient.h

```diff
@@ -206,6 +207,19 @@ namespace PVR
     std::vector<std::pair<std::string, int>> m_recordingsLifetimeValues;
   };
 
+  class CPVRClientProvider
```

Comment:
Put this class in its own header file, please.

Reply:
Is separated in own files